src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Models/FileSystemCmdletProviderEncoding.cs

```diff
@@ -12,24 +12,23 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.Dns.Services;
```

Comment:
string diff

Reply:
This seems to be some kind of GitHub diff issue. Looking at the Git log for the file, there are only 2 commits in the history. 1 for creating it and another when I added the comment linking to the GitHub issue I created for it.